packages/components/src/tools-panel/types.ts

```diff
@@ -16,6 +16,12 @@ export type ToolsPanelProps = {
 	 * The child elements.
 	 */
 	children: ReactNode;
+	/**
+	 * The popover props to configure panel's DropdownMenu.
+	 *
+	 * @default {}
+	 */
+	dropdownMenuProps?: {};
```

Comment:
The correct type for `dropdownMenuProps` should be "the props that the `DropdownMenu` component accepts.

So, something like `React.ComponentProps< typeof DropdownMenu >` should work. The thing to consider is whether we should `Omit` any of those props (for example, `children`).

Reply:
Ah, I see. I've updated via https://github.com/WordPress/gutenberg/pull/55785/commits/ea682b935b9bfb9d269133ebacb240c339c15cae.

Reply:
> The thing to consider is whether we should Omit any of those props (for example, children).

What do you think?

Reply:
We probably don't want to accept `children`. It would also make little sense to override props like `icon` and `label`, but I wonder if we should just allow that for the sake of extensibility. I'll let @aaronrobertshaw advise on this one, since he's more experienced with `ToolsPanel` than I am.

Related note: I guess that we drop the `dropdownMenuClassName` prop from `ToolsPanelHeader`, and use `dropdownMenuProps` directly instead?